CMakeLists.txt

```diff
@@ -230,12 +236,16 @@ set(arbor_supported_components)
 
 # Target microarchitecture for building arbor libraries, tests and examples
 #---------------------------------------------------------------------------
-if(ARB_ARCH)
+if(ARB_ARCH STREQUAL "none")
+    set(ARB_CXX_FLAGS_FULL "${ARB_CXX_FLAGS}")
+else(ARB_ARCH)
```

Comment:
if we check against empty string above, we don't need to check here again (also cf. special case handling the string "OFF" evaluates to false.

Reply:
I remove the `else(ARB_ARCH)`, because that was a cut and paste error.